ez/ez.go

```diff
@@ -16,6 +16,33 @@ import (
 	"github.com/vimeo/dials/yaml"
 )
 
+// Option sets an option on dialsOptions
+type Option func(*dialsOptions)
+
+type dialsOptions struct {
+	watch      bool
+	flagConfig *flag.NameConfig
+}
+
+func getDefaultOption() *dialsOptions {
+	return &dialsOptions{
+		flagConfig: flag.DefaultFlagNameConfig(),
+	}
```

Comment:
nit: Let's initialize `watch` to `false` explicitly.

Reply:
Done